vm/src/pyobjectrc.rs

```diff
@@ -255,37 +516,69 @@ impl PyObjectRef {
 
 impl PyObject {
     #[inline]
-    fn with_pyobjectref<R>(&self, f: impl FnOnce(&PyObjectRef) -> R) -> R {
-        // SAFETY: we don't use obj after the real lifetime (self) goes out of scope, we wrap it
-        //         in a ManuallyDrop to prevent double free, and we only pass it by reference
-        let obj = unsafe { ManuallyDrop::new(PyObjectRef::from_raw(self)) };
-        f(&obj)
+    fn weakreflist(&self) -> Option<&WeakRefList> {
```

Comment:
```suggestion
    fn weak_ref_list(&self) -> Option<&WeakRefList> {
```
unless it is a python symbol

Reply:
Yea it is a python symbol, grepping for `weakreflist` in cpython gives 170 matches

Reply:
I don't think the 170 matches are used as python symbol. Most of them are used as just CPython inner names. They are not exposed as `weakreflist` in Python code